### PR TITLE
update swagger spec

### PR DESCRIPTION
**Pull request by ElvisChiang**

### CODE REVIEW

File: pkg/api/api.go [hunk 1]
```diff
@@ -29,7 +29,6 @@
 //          in: header
 //
 // swagger:meta
-
```
Comment (ElvisChiang) [inline pkg/api/api.go]: the blank line is not accepted by go-swagger, have to remove it

File: assets/swaggerui/swagger.json [hunk 2]
```diff
@@ -302,120 +338,19 @@
...
       "description": "AccessRole represents access role type",
       "type": "integer",
-      "format": "int8",
-      "x-go-package": "github.com/ribice/gorsk/internal"
+      "format": "int64",
+      "x-go-package": "github.com/ribice/gorsk/pkg/utl/model"
     },
-    "Credentials": {
-      "description": "Credentials contains login request",
+    "HTTP": {
```
Comment (ElvisChiang) [inline assets/swaggerui/swagger.json]: maybe need a `swagger:ignore` in code
`pkg/api/password/transport` and `pkg/api/user/transport`

File: swagger.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+swagger generate spec -o assets/swaggerui/swagger.json --scan-models -b github.com/ribice/gorsk/pkg/api
```
Comment (ribice) [inline swagger.sh]: `swagger generate spec -o ./swagger.json --scan-models`

This one works fine for me (and is shorter).

Comment (ElvisChiang) [reply swagger.sh]: cd $GOPATH/src/github.com/ribice/gorsk

My go-swagger will generate empty swagger.json without specify base directory (-b)

Comment (ribice) [reply swagger.sh]: I run it inside `cmd/api`, not `pkg/api`. Please update your PR to include`swagger.json`, then I can merge it. Thanks!

Comment (ElvisChiang) [reply swagger.sh]: Do you mean putting an extra swagger.json in `cmd/api/swagger.json`  besides `assets/swaggerui/swagger.json`?

**Comment by ribice** [comment]
Can you remove all files except for `swagger.json` from PR? That should be it.

**Pull request merged**